omero/autogen_docs

```diff
@@ -12,7 +12,11 @@ WORKSPACE=${WORKSPACE:-$(pwd)}
 USER=${USER:-$(whoami)}
 
 echo "Copying history"
-cp $WORKSPACE/OMERO.server/history.rst omero/users/history.rst
+# inject link to change log
+touch newfile.rst
+cat omero/changelog.rst > $WORKSPACE/newfile.rst
```

Comment:
Why `$WORKSPACE` here but not for the `touch`?

Reply:
mistake!